spec/models/post.spec.js

```diff
@@ -10,11 +10,16 @@ describe("Post model", () => {
     });
   });
 
-  it("has a message", () => {
+  it("post has a message", () => {
```

Comment:
possibly more test ?

Reply:
I changed the description on this test because the post now has a message and a date, as previously only had a message